src/main/webapp/chart.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script
+      type="text/javascript"
+      src="https://www.gstatic.com/charts/loader.js"
+    ></script>
+    //TODO move all of this to a separate JavaScript file
+    //figure out a way to get user input/data for the charts
+    //make sure to implement same idea as in the stats page
+    <script type="text/javascript">
+      google.charts.load("current", { packages: ["corechart"] });
```

Comment:
I recommend wrapping your JavaScript code in a function to prevent leakage of symbols into the global scope as in:

    (function(){
       var drawChart = function() {
          // ...
       };
 
       var init = function() {
         google.charts.load('current', {  packages: ['corechart'] });
         google.charts.setOnLoadCallback(drawChart);
       };

       init();
    })();

The outer `function(){ ... }` creates a scope that prevents the symbols from becoming global, while the `();` at the end invokes the anonymous function.